### PR TITLE
[#761] Pin all third-party docker images with digest sha

### PR DESCRIPTION
**Pull request by mcdoker18**

### CODE REVIEW

**Comment by mcdoker18** [comment]
@kirillmakhonin fixed

**Comment by mcdoker18** [comment]
test, build all

**Comment by mcdoker18** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test all

File: legion/tests/data/empty-docker/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
-FROM alpine:3.5
+FROM ubuntu:18.04@sha256:945039273a7b927869a07b375dc3148de16865de44dec8398672977e050a072e
```
Comment (aliaksandr-d) [inline legion/tests/data/empty-docker/Dockerfile]: Excuse me, why was it switched to ubuntu?

**Comment by mcdoker18** [comment]
bot, test all

File: legion/tests/data/empty-docker/Dockerfile (hunk 1 above)
Comment (mcdoker18) [reply legion/tests/data/empty-docker/Dockerfile]: Opps, fixed

**Comment by mcdoker18** [comment]
bot, test all

**Pull request merged**